kuzzleio/kuzzle
# Add action and controller in request input for http events

## What does this PR do ?

This PR add `request.input.action` and `request.input.controller` in the RequestInput when triggering `http:*` events.

Fix https://github.com/kuzzleio/kuzzle/issues/1142

### How should this be manually tested?

  - Step 1 : Checkout the branch `git fetch origin 1142-fix-request-input-on-http-event && git checkout 1142-fix-request-input-on-http-event`
  - Step 2 : Modify the logger plugin `plugins/enabled/kuzzle-plugin-logger/lib/index.js` and add a hook that log `request.input.action` and `request.input.controller` on `http:get` 
  - Step 3 : Check with `curl http://localhost:7512/users/_me`

### Boyscout

 -  remove usage of useless sandbox
